### PR TITLE
[45] Product Page using HTML - Fullstack

### PR DESCRIPTION
**Pull request by ToluEnny24**
I created a product page and added the images

implementation page

### CODE REVIEW

File: submissions/fabunmiTolulope/HTML/Productpage.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+				<li><a href="aboutUs.html">Contact Us</a> </li>
+			</ul>
+		</div>
+	</nav>
+	<hr>
+<section>
+	<H3> Organic Giant Cavendish Banana</H3>
+	<img src="cavendish banana.jpg"; height="200px" width="200px";>
+	<img src="cavendish.jpg"; height="200px" width="200px";>
+	<p align="left"><strong> &#x20A6;10,000</strong></p>
```
Comment (emmanuelidiaghe) [inline submissions/fabunmiTolulope/HTML/Productpage.html:28]: Also remove the align attribute of the p tag. CSS would take care of that later. For a more seamless review process, please leave all styling for CSS.

**Review by ainobs — CHANGES_REQUESTED** [review]

**Review by emmanuelidiaghe — CHANGES_REQUESTED** [review]
Please address these issues- h3, styling, etc

**Review by ainobs — APPROVED** [review]

File: submissions/fabunmiTolulope/HTML/Productpage.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+<body>
+<header>
+		<h1>Welcome to Greenprint Integrated Services</h1>
+</header>
+	<nav>
+		<ul>
+				<li><a href="home.html"> Home</a></li>
+				<li><a href="blog.html">Blog</a> </li>
+				<li><a href="Productpage.html">Productpage</a> </li>
+				<li><a href="aboutUs.html">Contact Us</a> </li>
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/HTML/Productpage.html:18]: There is no `aboutUs.html` on your project. Can you remove it?

**Review by adaugochi — CHANGES_REQUESTED** [review]
Also, remove spaces on images names. Replace with either `_` or `-`.

Place your images inside an `images` folder inside your `HTML` folder

File: submissions/fabunmiTolulope/HTML/Productpage.html (hunk 2 above)
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/HTML/Productpage.html]: Yes, i have removed it

File: submissions/fabunmiTolulope/HTML/Productpage.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	</ol> 
+</p><hr>
+</section>
+<nav>
+	<ul>
+		<li><a href="#"> Add to cart</a></li>
+		<li><a href="#">View more products</a> </li>
+	</ul>
+</nav>
+<FOOTER>
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/HTML/Productpage.html:47]: This should be in lowercase. We want consistency

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/fabunmiTolulope/HTML/Productpage.html (hunk 3 above)
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/HTML/Productpage.html:47]: i will effect that immediately

Comment (ToluEnny24) [reply submissions/fabunmiTolulope/HTML/Productpage.html]: lowercase now

File: .gitignore [hunk 4]
```diff
@@ -3,3 +3,4 @@
 
 # PhpStorm Project Files
 .idea
+submissions/FabunmiTolulopeEniola/
```
Comment (adaugochi) [inline .gitignore:6]: Why placing this file path in `.gitignore` file?

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 4 above)
Comment (ToluEnny24) [reply .gitignore:6]: I could not delete it

**Comment by ToluEnny24** [comment]
i have deleted it on my local and remote repo i believe.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**